docs/guides/anomaly-detection-configuration/training-period.mdx

```diff
@@ -0,0 +1,69 @@
+---
+title: "training_period"
+sidebarTitle: "training_period"
+---
+
+```
+training_period:
+  period: < time period > # supported periods: day, week, month
+  count: < number of periods >
+```
+
+The maximal timeframe for which the test will collect data.
+This timeframe includes the training period and detection period.
```

Comment:
Does this timeframe also include the detection delay? It's worth writing here if yes or no

Reply:
yes, added.